client/src/components/common/names-capture.vue

```diff
@@ -652,25 +655,25 @@ export default class NamesCapture extends Vue {
     this.validate(true)
   }
   validatePhrases (choice: string) {
-    if (!this.editMode) {
-      let name = this.nameChoices[choice]
-      if (this.entityPhraseRequired) {
-        let name = this.nameChoices[choice].toUpperCase()
-        if (this.entityPhraseChoices.every(phrase => {
-          phrase = phrase.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&')
-          return (name.search(new RegExp('(\\s)' + phrase + '(\\s|$)')) === -1)
-        })) {
-          this.messages[choice] = 'Your name must contain one of the required phrases'
-        }
-        if (this.entityPhraseChoices.some(phrase => name.startsWith(phrase))) {
-          this.messages[choice] = 'Your name must not begin with one of the required phrases'
-        }
+    if (this.entityPhraseRequired) {
+      const name = this.nameChoices[choice].toUpperCase()
+      if (this.entityPhraseChoices.every(phrase => {
+        phrase = phrase.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&')
+        return (name.search(new RegExp('(\\s)' + phrase + '(\\s|$)')) === -1)
+      })) {
+        this.messages[choice] = `A ${this.entityTypeText} name must include (but not start with)
+          one of the following phrases: ${this.entityPhraseText}`
+      }
+      if (this.entityPhraseChoices.some(phrase => name.startsWith(phrase))) {
+        this.messages[choice] = `A ${this.entityTypeText} name must not begin with one of the
+          following phrases: ${this.entityPhraseText}`
       }
     }
   }
 }
 </script>
 
-<style lang="sass">
-
+<style scoped lang="sass">
+::v-deep .v-messages__message
+  line-height: 14px !important
```

Comment:
Line height _was_ 12px, same as the default font size here, but the bottom was cut off (ie, chars that dangled, like "g" and "y").